src/components/application_manager/src/state_controller_impl.cc

```diff
@@ -245,129 +245,185 @@ void StateControllerImpl::SetRegularState(ApplicationSharedPtr app,
 }
 
 namespace {
-bool HasStreamingPermissions(mobile_apis::HMILevel::eType val) {
+bool IsStreamableHMIType(mobile_apis::HMILevel::eType val) {
```

Comment:
@EKuliiev not consistant naming. You check that IsStreamableHMI**Type**, but pass level as parameter. There is separate  essence named HMIType and it is confusing.

Reply:
@alexkutsan see c7da899496a38e14f81e91e02389613313af62e8

Reply:
@EKuliiev Who is Alexkutsan?